presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -16,10 +16,13 @@
 import io.airlift.configuration.Config;
 import io.airlift.configuration.ConfigDescription;
 
+import javax.validation.constraints.Min;
+
 public class JdbcMetadataConfig
 {
     private boolean allowDropTable;
     private boolean allowAggregationPushdown = true;
+    private int domainSizeThreshold = 10_000;
```

Comment:
sounds like a lot. Based on the limitations in oracle, sql server, wouldn't 500 be a safer number?